tests/src/RSS/ArticleParserTest.php

```diff
@@ -38,14 +38,21 @@ public function testParseParagraph()
         $text = $parser->parseParagraph('<b>some other html before</b><p class="medium-feed-snippet">Find me?</p><p>some other paragraph after</p>');
         $this->assertEquals('Find me?', $text);
 
-        $text = $parser->parseParagraph('<p class="medium-feed-snippet">Find me?</b>');
-        $this->assertEquals('Find me?', $text, 'Parser makes best guess with invalid input');
-
         $text = $parser->parseParagraph('<p class="medium-feed-snippet">Find me?');
         $this->assertEquals('Find me?', $text, 'Parser makes best guess with invalid input');
 
         $text = $parser->parseParagraph('<b>no paragraph</b>');
         $this->assertEquals('', $text);
+
+        $text = $parser->parseParagraph('<p>no class on paragraph</p>');
+        $this->assertEquals('no class on paragraph', $text);
+
+        // I'm not sure why this works.
+        $text = $parser->parseParagraph('<p>no class on paragraph<p>Nested paragraph</p></p>');
```

Comment:
That's what the test seems to imply, but I really don't know how it works. Nested paragraphs are invalid HTML, so the case will be rare, and probably best to avoid on Medium